lib/scout_apm/auto_instrument/instruction_sequence.rb

```diff
@@ -0,0 +1,22 @@
+
+require 'scout_apm/auto_instrument/rails'
+
+module ScoutApm
+  module AutoInstrument
+    module InstructionSequence
+      CONTROLLER_PATH_PATTERN = /\/apm\/app\/controllers\/.*_controller.rb$/
+
+      def load_iseq(path)
+        if path =~ CONTROLLER_PATH_PATTERN
+          new_code = Rails.rewrite(path)
+
+          return compile(new_code, path, filepath)
+        else
+          return super
```

Comment:
Forgot this crucial one: we must assume `super` (`load_iseq`) does not exist (but it *might* exist).

Reply:
Yeah, I reworked it to be `compile_file`. It's simple solution for now.

Reply:
We could probably write `super rescue compile_file(path)` but that might be a bad idea.